scripts/nightly-version.js

```diff
@@ -20,16 +20,14 @@ const currentRev = childProcess // get the current rev from the repo
     .execSync('git rev-parse HEAD')
     .toString()
     .trim()
-    .slice(0, 7)
-;
+    .slice(0, 7);
```

Comment:
@mvorisek This rule change is inconsistenst (at several places)
You changed the one-var rule so each line could be easier deleted/added without touching another line to change the end "," to ";" or vice versa.

The same argument could be applied here too: Forcing the semicolon to the last method call here would need a change to that line if i ever want to add another methode to the string chain.

So, when the argument for the one-var rule is "i change change the single line without touching others", i think this should be applied here too, so the last semicolon should be kept in a single line as before (So i would be able to delete/add it without touching any other line

I won't continue reviewing until this is adjusted/discussed

Reply:
This is true and I am aware of it.

The only negative is extra line in diff when the last chained call is removed/added.

IMO long-term-stable CS outperforms this negative.

Given this negative is only one of many reasons of this PR and given there is no `semi-style` rule allowing us placing semicolon on a separate line, we should do a compromise and use `semi-style: last`.

This config is also default in default esling config, AirBnB and Unicorn stylesets, so let's land it and have the CS normalized and asserted by CI.